test/test_basic.py

```diff
@@ -34,7 +34,7 @@ async def test_rsa_key(self):
             await session.connect(server.host,
                                   port=server.port,
                                   configfile='none',
-                                  options={'UserKnownHostsFile': '/dev/null'},
+                                  options={'UserKnownHostsFile': '/dev/null', 'StrictHostKeyChecking': 'no'},
```

Comment:
I wonder if those deserve to be on separate lines (with an extra indent) at this point...

Reply:
It's "just" 109 chars on that line, but I don't mind. It looks nicer as separate lines. Done.